specification/cognitiveservices/OpenAI.Inference/models/images.tsp

```diff
@@ -0,0 +1,137 @@
+import "@azure-tools/typespec-azure-core";
+import "@typespec/http";
+
+using Azure.Core;
+using TypeSpec.Http;
+
+namespace Azure.OpenAI;
+
+@doc("The desired size of the generated images. Must be one of 256x256, 512x512, or 1024x1024.")
+enum ImageSize {
+  @doc("Small image size (256x256).")
+  Small: "256x256",
+
+  @doc("Medium image size (512x512).")
+  Medium: "512x512",
+
+  @doc("Large image size (1024x1024).")
+  Large: "1024x1024",
+}
+
+@doc("Represents the request data used to generate images.")
+model ImageRequest {
+  @doc("A description of the desired images.")
+  prompt: string;
+
+  @doc("The number of images to generate (defaults to 1).")
+  n?: int32;
+
+  @doc("The desired size of the generated images. Must be one of 256x256, 512x512, or 1024x1024 (defaults to 1024x1024).")
```

Comment:
The comment does not match the enum value names. Which can be solved by not renaming the values in the enum.

Reply:
changed the enum values back to the sizes